.github/workflows/pr.yaml

```diff
@@ -0,0 +1,79 @@
+on: pull_request
+name: Review
+jobs:
+  'assets-coding-standards-pratchett':
+    name: Check assets coding standards (pratchett)
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        node: ['10']
+    steps:
+      - uses: actions/checkout@v2
+      - name: Setup node
+        uses: actions/setup-node@v2
+        with:
+          node-version: ${{ matrix.node }}
+      - run: |
+          yarn --cwd sites/all/themes/pratchett/ install
```

Comment:
You should add cache to yarn to speed up tests (we have limit min. pr. month in actions with this free account).

For details see https://github.com/actions/cache 

```
- uses: actions/cache@v2
  with:
    path: '**/node_modules'
    key: ${{ runner.os }}-modules-${{ hashFiles('**/yarn.lock') }}
```

Please use this with all `install` of assets.

Reply:
Done.